index.d.ts

```diff
@@ -58,6 +58,7 @@ declare module 'react-native-inappbrowser-reborn' {
     hasBackButton?: boolean,
     browserPackage?: string,
     showInRecents?: boolean
```

Comment:
@vincent-paing Oh a comma is missing here, I didn't see that! 😮

Reply:
Ah, my bad. Have you patched this?

Reply:
Sure thing, I used Prettier to fix the format of this file, thanks!